app/ante.go

```diff
@@ -65,40 +111,48 @@ func (ad IsBlacklistedDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate
 			switch m := m.(type) {
 			case *banktypes.MsgSend:
 				for _, c := range m.Amount {
-					if checkIfMintedAsset(ctx, c.Denom, ad.tokenfactory) {
-						addresses = append(addresses, m.ToAddress, m.FromAddress)
+					addresses = append(addresses, m.ToAddress, m.FromAddress)
```

Comment:
```suggestion
					addresses := []string{m.ToAddress, m.FromAddress}
```

Reply:
we can do this (same for the below also) and remove `var addresses []string` to prevent any addresses from being checked more than once and make it more clear that only the specified address(es) will be checked.

Reply:
Nice! Implemented.